pages/in/widget/iscn/dev/index.vue

```diff
@@ -3,72 +3,129 @@
   <div
     v-if="isLoading"
     key="loading"
-    class="likepay-body likepay-body--center"
+    class="likepay-body likepay-body--center iscn-body"
   >
     <span class="likepay-text-panel">{{ $t('General.loading') }}</span>
   </div>
   <div
     v-else
     key="panel"
-    class="likepay-body"
+    class="likepay-body iscn-body"
   >
+    <header class="iscn-body__header">
+      <svg
+        xmlns="http://www.w3.org/2000/svg"
+        viewBox="0 0 30 30"
+        width="30"
+      >
+        <path
+          :d="`
+            M20.16,9.55,15.58,5
+            C11.22.61,2.42-1.06.68.68
+            S.61,11.22,5,15.58
+            l4.58,4.58
+            a12.29,12.29,0,0,0,2.69,9.44,1.09,1.09,0,0,0,1.6.09
+            l2.62-2.62,1.83-1.83,1.55.91,6.3-6.3-.91-1.55,1.83-1.83,2.62-2.62
+            a1.08,1.08,0,0,0-.1-1.6,12.3,12.3,0,0,0-9.43-2.7
+          `"
+          fill="currentColor"
+          fill-rule="evenodd"
+        />
+      </svg>
+      <h1>{{ $t('ISCNWidget.title') }}</h1>
+    </header>
+
     <div class="likepay-panel">
       <section class="likepay-panel__section-container">
-        <header class="likepay-panel__section-header">
-          <div class="likepay-panel__header-title">{{ $t('ISCNWidget.title') }}</div>
-        </header>
+
         <div class="likepay-panel__section-meta">
-          <div class="likepay-panel__section-meta-grid-item-label">
+          <div class="likepay-panel__section-meta-label">
             {{ $t('ISCNWidget.label.title') }}
           </div>
-          <div class="likepay-panel__section-meta-grid-item-value">{{ title }}</div>
-          <div v-if="getUserInfo" class="likepay-panel__section-meta">
-            <div class="likepay-panel__section-meta-label">{{ $t('ISCNWidget.label.Creator') }}</div>
+          <div class="likepay-panel__section-meta-value">{{ title }}</div>
+        </div>
+
+        <div
+          v-if="getUserInfo"
+          class="likepay-panel__section-meta"
+        >
+          <div class="likepay-panel__section-meta-label">
+            {{ $t('ISCNWidget.label.creator') }}
+          </div>
+          <a
+            :href="`${LIKER_LAND_URL}/${getUserInfo.user}`"
+            target="_blank"
+            rel="noopener"
+            class="likepay-panel__user"
+          >
             <lc-avatar
+              v-if="getUserInfo.avatar"
               :src="getUserInfo.avatar"
-              :halo="avatarHalo"
```

Comment:
Should the `avatarHalo` property below be removed also?

Reply:
Seem rebased mistake